examples/Microsoft.Spark.CSharp.Examples/MachineLearning/Sentiment/Program.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.MachineLearning.Sentiment
+{
+    /// <summary>
+    /// Example of using ML.NET + .NET for Apache Spark
+    /// for sentiment analysis.
+    /// </summary>
+    //public class Program
+    internal sealed class Program : IExample
+    {
+        public void Run(string[] args)
+        {
+            if (args.Length != 1)
```

Comment:
Instead of a path to the Resources folder, can we instead have the user enter in the full path to the ML Model zip and also the full path to the dataset to run the ML model against ?

Reply:
Sure, I can make that change. I had written it this way because I thought it might make it easier to have fewer command line arguments. Is your thought here that it'd be better to have the user enter the two paths so that they aren't restricted to the Resources folder/requiring both resources to be in the same folder?

Reply:
If you do it how it is, then you'll have to use `Path.Combine()` for yelptest.csv and MLModel.zip

Reply:
Makes sense. Changed, thanks!

Reply:
Can you update the readme too